comtypes/tools/codegenerator.py

```diff
@@ -1274,6 +1165,108 @@ def make_DispProperty(self, prop):
 
         print(code, file=self.stream)
 
+
+class ImportedNamespaces(object):
+    def __init__(self):
+        if sys.version_info >= (3, 7):
+            self.data = {}
+        else:
+            from collections import OrderedDict
+            self.data = OrderedDict()
+
+    def add(self, *names, symbols=None):
```

Comment:
This syntax (`*names` and further keyword args) is not Py2.7 compatible. I still want to keep it for pywinauto project.

Reply:
I made this py27-compatible.

P.S.
`*names, symbols=None` syntax was not failed in AppVeyor CI.
Perhaps, some tests are broken.

IMO release of 1.1.13 should wait until tests were fixed.

Reply:
`comtypes` AppVeyor tests are very limited. There are only 2 install tests. The vast majority of functional tests are not enabled in AppVeyor for now. I've detected this Py2.7 issue on pywinauto tests (on pywinauto's AppVeyior). If they pass, I'm probably going to merge this PR. Thanks for your efforts!